magic-link/magic-link-templates.php

```diff
@@ -1293,6 +1297,12 @@ public function update_record( WP_REST_Request $request ) {
             }
         }
 
+        // Next, dispatch submission notification, accordingly; always send by default.
+        if ( $params['send_submission_notifications'] && isset( $updated_post['assigned_to'], $updated_post['assigned_to']['id'], $updated_post['assigned_to']['display'] ) ) {
+            $submission_comment = '@[' . $updated_post['assigned_to']['display'] . '](' . $updated_post['assigned_to']['id'] . ') ' . $params['send_submission_notifications_comment'];
```

Comment:
@kodinkat lets build the comment here instead of passing the text to the front end to be passed back.